### PR TITLE
Add Matt Craig to lifetime contributors

### PR DESCRIPTION
**Pull request by hamogu**
This is based on answers to the 2022/01 roles survey.

### CODE REVIEW

File: credits.html [hunk 1]
```diff
@@ -159,6 +159,14 @@ <h3>Marten van Kerkwijk</h3>
     accuracy at the level needed for pulsar timing, and he played an important
     role in making the Table class versatile.
 
+    <h3>Matt Craig</h3>
+    Since becoming involved in the project in fall 2013, Matt co-lead the
```
Comment (mwcraig) [inline credits.html:163]:
```suggestion
    Since becoming involved in the project in fall 2013, Matt co-led the
```

File: credits.html [hunk 2]
```diff
@@ -159,6 +159,14 @@ <h3>Marten van Kerkwijk</h3>
...
+    development of ccdproc, the coordinated package for optical/IR image data
+    reduction and served as lead for the NDData subpackage. He also served
+    on multiple Python in Astronomy organizing committee, helped with a couple
```
Comment (mwcraig) [inline credits.html:166]:
```suggestion
    on multiple Python in Astronomy organizing committees, helped with a couple
```

**Review by mwcraig — COMMENTED** [review]
Looks good, just a couple littles typos I noticed.

**Review by pllim — APPROVED** [review]
👏

**Pull request merged**